lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -308,7 +308,12 @@ export class RefreshTokenClient extends BaseClient {
             acquireTokenMeasurement?.end({
                 success: true,
             });
-            return result;
+
+            if (!response) {
```

Comment:
How can this happen? What is the error condition that the user / app dev is facing and how can they fix it? Errors should be actionable.

Reply:
Reverted these changes. I was messing around in this file while testing the unit tests.

Reply:
It will never happen; I made a mistake leaving that code in this PR. There will always either be a response, or a thrown exception.